magenta/music/musicxml_parser.py

```diff
@@ -110,12 +113,26 @@ def get_score(filename):
       namelist = filename.namelist()
       files = [name for name in namelist if not name.startswith('META-INF/')]
       compressed_file_name = files[0]
-      score = ET.fromstring(filename.read(compressed_file_name))
+      try:
+        score = ET.fromstring(filename.read(compressed_file_name))
+      except ET.ParseError as parse_error:
+        line_number, column = parse_error.position
+        line = next(IT.islice(io.BytesIO(content), line_number))
+        caret = '{:=>{}}'.format('^', column)
+        parse_error.msg = '{}\n{}\n{}'.format(parse_error, line, caret)
+        print(parse_error.msg)
```

Comment:
Rather than printing the parse error, it would be better to raise a custom exception that contains the parse error, so the caller can decide what to do with the error (e.g., skip the file silently, print out an error, add it to an error log, etc.).

Wrapping the ET.ParseError exception in a custom exception is important because callers into this class shouldn't need to know that this class uses the ElementTree class. They should just need to know how to handle exceptions that this class might raise.